db/migrations/61_make_users_items_active_attempt_id_not_null.sql

```diff
@@ -0,0 +1,30 @@
+-- +migrate Up
+ALTER TABLE `users_items`
+    MODIFY COLUMN `active_attempt_id` bigint(20) NOT NULL COMMENT 'Current attempt selected by this user.',
+    DROP INDEX `active_attempt_id`,
+    DROP PRIMARY KEY,
+    DROP COLUMN `id`,
+    DROP INDEX `user_item`,
+    ADD PRIMARY KEY `fk_user_id_item_id` (`user_id`, `item_id`),
+    ADD FOREIGN KEY `fk_user_id` (`user_id`) REFERENCES `users`(`id`) ON DELETE CASCADE,
+    ADD FOREIGN KEY `fk_item_id` (`item_id`) REFERENCES `items`(`id`) ON DELETE CASCADE,
+    ADD FOREIGN KEY `fk_active_attempt_id` (`active_attempt_id`) REFERENCES `groups_attempts` (`id`) ON DELETE CASCADE;
```

Comment:
I don't see the logic behind `(active_attempt_id, item_id)` since `active_attempt_id` is unique

Reply:
In a nutshell: Otherwise you write a users_items with an `item_id` not matching the one of the `active_attempt_id` and vice-versa. A valid alternative of course would be to remove `item_id` as it is implicit in the active attempt.

Reply:
Can we think about this later (when we'll have FKs in groups_attempts)?

Reply:
Yes we can 🙂